com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/variables/VariableProxy.java

```diff
@@ -30,10 +31,11 @@
      * @param variable
      *              the variable object
      */
```

Comment:
I find you only use the depth of the stackFrame, use a depth instead?

Reply:
nice.